clinvar/src/main/scala/org/broadinstitute/monster/etl/clinvar/models/output/SCV.scala

```diff
@@ -52,7 +53,8 @@ case class SCV(
   submittedAssembly: Option[String],
   interpretationDescription: Option[String],
   interpretationLastEvaluated: Option[String],
-  interpretationComments: Array[String]
+  interpretationComments: Array[String],
+  rcvId: Option[String]
```

Comment:
Can you also add a column for `traitSetId`? It's easy to reconstruct, but I think ClinGen wants it there.